app/src/main/java/com/example/movopfy/features/title/presentation/ui/TitleScreen.kt

```diff
@@ -1,62 +1,60 @@
-package com.example.movopfy.features.title.presentation
+package com.example.movopfy.features.title.presentation.ui
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyColumn
-import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.font.Font
-import androidx.compose.ui.text.font.FontFamily
-import androidx.compose.ui.unit.dp
-import androidx.compose.ui.unit.sp
 import coil.compose.AsyncImage
-import com.example.movopfy.R
-import com.example.movopfy.main.ui.theme.BackgroundMain
-import com.example.movopfy.main.ui.theme.Dimensions
-import com.example.movopfy.main.ui.theme.LocalDim
-import com.example.movopfy.main.ui.theme.TextMain
-import com.example.movopfy.uiComponents.ProgressBarLoading
+import com.example.movopfy.common.extensions.getSmallImageUrl
+import com.example.movopfy.features.title.presentation.viewmodel.TitleViewModel
+import com.example.movopfy.uiComponents.components.ProgressBarLoading
+import com.example.movopfy.uiComponents.theme.BackgroundMain
+import com.example.movopfy.uiComponents.theme.LocalDim
 import org.koin.androidx.compose.koinViewModel
 
 @Composable
 fun TitleScreen(viewModel: TitleViewModel = koinViewModel(), id: Int) {
```

Comment:
Please rename package and this file from Title to Details. It will be `DetailsScreen`, `details` package